supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/callcaching/AwsBatchBackendCacheHitCopyingActor.scala

```diff
@@ -47,53 +48,78 @@ import scala.language.postfixOps
+          }
       }
+    }
+    case _ => {
+      super.processSimpletons(womValueSimpletons, sourceCallRootPath)
+    }
   }
 
-  override def processDetritus(sourceJobDetritusFiles: Map[String, String]) = cachingStrategy match {
-    case CopyCachedOutputs => super.processDetritus(sourceJobDetritusFiles)
-    case UseOriginalCachedOutputs =>
-      // apply getPath on each detritus string file
-      val detritusAsPaths = detritusFileKeys(sourceJobDetritusFiles).toSeq map { key =>
-        key -> getPath(sourceJobDetritusFiles(key))
-      } toMap
+  override def processDetritus(sourceJobDetritusFiles: Map[String, String]) = batchAttributes.fileSystem match {
+    case "s3" => {
+      cachingStrategy match {
+        case CopyCachedOutputs => super.processDetritus(sourceJobDetritusFiles)
+        case UseOriginalCachedOutputs =>
+          // apply getPath on each detritus string file
+          val detritusAsPaths = detritusFileKeys(sourceJobDetritusFiles).toSeq map { key =>
+            key -> getPath(sourceJobDetritusFiles(key))
+          } toMap
 
-      // Don't forget to re-add the CallRootPathKey that has been filtered out by detritusFileKeys
-      TryUtil.sequenceMap(detritusAsPaths, "Failed to make paths out of job detritus") map { newDetritus =>
-        (newDetritus + (JobPaths.CallRootPathKey -> destinationCallRootPath)) -> newDetritus.values.map(S3BatchCommandBuilder.touchCommand).toSet
+          // Don't forget to re-add the CallRootPathKey that has been filtered out by detritusFileKeys
+          TryUtil.sequenceMap(detritusAsPaths, "Failed to make paths out of job detritus") map { newDetritus =>
+            (newDetritus + (JobPaths.CallRootPathKey -> destinationCallRootPath)) -> newDetritus.values.map(S3BatchCommandBuilder.touchCommand).toSet
+          }
       }
+    }
+    case _ => {
+      super.processDetritus(sourceJobDetritusFiles)
+    }
   }
 
+
   override protected def additionalIoCommands(sourceCallRootPath: Path,
                                               originalSimpletons: Seq[WomValueSimpleton],
                                               newOutputs: CallOutputs,
                                               originalDetritus:  Map[String, String],
                                               newDetritus: Map[String, Path]): List[Set[IoCommand[_]]] = {
-    cachingStrategy match {
-      case UseOriginalCachedOutputs =>
-        val content =
-          s"""
-             |This directory does not contain any output files because this job matched an identical job that was previously run, thus it was a cache-hit.
-             |Cromwell is configured to not copy outputs during call caching. To change this, edit the filesystems.aws.caching.duplication-strategy field in your backend configuration.
-             |The original outputs can be found at this location: ${sourceCallRootPath.pathAsString}
+    batchAttributes.fileSystem match {
```

Comment:
It is possible that a pattern like
```
(batchAttributes.fileSystem, cachingStrategy) match {
  case ("s3", UseOriginalCachedOutputs) => ???
}
```
may be appropriate

Reply:
Done!